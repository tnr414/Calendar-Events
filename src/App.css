  .left { float: left; }
  
  body {
      padding:0;
  }
  
  #wrapper {
      width:800px;
      margin: 10px auto;
  }
  
  #calander {
      width:600px;
      height:720px;
      padding: 0 10px;
      margin:0 auto;
      background:#ECECEC;
      position:relative;
      overflow:hidden;
  }
  
  
  #times {
      height:720px;
      margin-right:10px;
      font-family:arial;
      margin-top:-3px;
  }
  
  #times .slot {
      height:30px;
      font-weight:bold;
      text-align:right;
      line-height:0;
  }
  
  #times .slot span {
      font-size: 12px;
      color:#999;
      padding:2px 0;
  }
  