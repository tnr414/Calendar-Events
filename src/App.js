import React from 'react'
import './App.css'
import { Schedular } from './components/Schedular/Schedular'

export const App = () => {
  return (
    <div data-testid = 'myapp'>
      <Schedular/>
    </div>
  )
}
