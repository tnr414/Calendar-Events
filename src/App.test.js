import { render, screen } from '@testing-library/react';
import { App } from './App'



test('the app', async() => {
  render(<App />);
  const element = screen.getByTestId(/myapp/i);
  expect(element).toMatchSnapshot();
});
