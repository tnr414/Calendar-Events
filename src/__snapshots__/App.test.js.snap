// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the app 1`] = `
<div
  data-testid="myapp"
>
  <div
    data-testid="schedular"
    id="wrapper"
  >
    <div
      class="left"
      data-testid="slotwrapper"
      id="times"
    >
      <div
        class="slot"
        data-testid="timeslot"
      >
        09:00
      </div>
      <div
        class="slot"
        data-testid="timeslot"
      >
        <span>
          09:30
        </span>
      </div>
      <div
        class="slot"
        data-testid="timeslot"
      >
        10:00
      </div>
      <div
        class="slot"
        data-testid="timeslot"
      >
        <span>
          10:30
        </span>
      </div>
      <div
        class="slot"
        data-testid="timeslot"
      >
        11:00
      </div>
      <div
        class="slot"
        data-testid="timeslot"
      >
        <span>
          11:30
        </span>
      </div>
      <div
        class="slot"
        data-testid="timeslot"
      >
        12:00
      </div>
      <div
        class="slot"
        data-testid="timeslot"
      >
        <span>
          12:30
        </span>
      </div>
      <div
        class="slot"
        data-testid="timeslot"
      >
        13:00
      </div>
      <div
        class="slot"
        data-testid="timeslot"
      >
        <span>
          13:30
        </span>
      </div>
      <div
        class="slot"
        data-testid="timeslot"
      >
        14:00
      </div>
      <div
        class="slot"
        data-testid="timeslot"
      >
        <span>
          14:30
        </span>
      </div>
      <div
        class="slot"
        data-testid="timeslot"
      >
        15:00
      </div>
      <div
        class="slot"
        data-testid="timeslot"
      >
        <span>
          15:30
        </span>
      </div>
      <div
        class="slot"
        data-testid="timeslot"
      >
        16:00
      </div>
      <div
        class="slot"
        data-testid="timeslot"
      >
        <span>
          16:30
        </span>
      </div>
      <div
        class="slot"
        data-testid="timeslot"
      >
        17:00
      </div>
      <div
        class="slot"
        data-testid="timeslot"
      >
        <span>
          17:30
        </span>
      </div>
      <div
        class="slot"
        data-testid="timeslot"
      >
        18:00
      </div>
      <div
        class="slot"
        data-testid="timeslot"
      >
        <span>
          18:30
        </span>
      </div>
      <div
        class="slot"
        data-testid="timeslot"
      >
        19:00
      </div>
      <div
        class="slot"
        data-testid="timeslot"
      >
        <span>
          19:30
        </span>
      </div>
      <div
        class="slot"
        data-testid="timeslot"
      >
        20:00
      </div>
      <div
        class="slot"
        data-testid="timeslot"
      >
        <span>
          20:30
        </span>
      </div>
      <div
        class="slot"
        data-testid="timeslot"
      >
        21:00
      </div>
    </div>
    <div
      class="left"
      data-testid="calendar"
      id="calander"
    />
  </div>
</div>
`;
